dali/operators/image/crop/crop_mirror_normalize.cu

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright (c) 2019-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 //
```

Comment:
What do we keep this one for?

Reply:
It's used to implement the ndim != 3 code path in the new one